app/models/game.rb

```diff
@@ -4,44 +4,47 @@ class Game < ActiveRecord::Base
   has_many :pieces
   scope :needing_second_player, -> { where(black_user_id: nil) }
 
+  
+
   def add_black_player(user)
     update(black_user: user)
+    initialize_board!
   end
 
   def initialize_board!
     # black pieces
     (0..7).each do |i|
-      Pawn.create(x_position: i, y_position: 1, game_id: id, user_id: black_user_id)
+      Pawn.create(x_position: i, y_position: 1, game_id: id, user_id: black_user_id, piece_unicode: "&#9823;")
     end
 
-    Rook.create(x_position: 0, y_position: 0, game_id: id, user_id: black_user_id)
-    Rook.create(x_position: 7, y_position: 0, game_id: id, user_id: black_user_id)
+    Rook.create(x_position: 0, y_position: 0, game_id: id, user_id: black_user_id, piece_unicode:"&#9820;")
+    Rook.create(x_position: 7, y_position: 0, game_id: id, user_id: black_user_id, piece_unicode:"&#9820;")
 
-    Knight.create(x_position: 1, y_position: 0, game_id: id, user_id: black_user_id)
-    Knight.create(x_position: 6, y_position: 0, game_id: id, user_id: black_user_id)
+    Knight.create(x_position: 1, y_position: 0, game_id: id, user_id: black_user_id, piece_unicode:"&#9822;")
+    Knight.create(x_position: 6, y_position: 0, game_id: id, user_id: black_user_id, piece_unicode:"&#9822;")
 
-    Bishop.create(x_position: 2, y_position: 0, game_id: id, user_id: black_user_id)
-    Bishop.create(x_position: 5, y_position: 0, game_id: id, user_id: black_user_id)
+    Bishop.create(x_position: 2, y_position: 0, game_id: id, user_id: black_user_id, piece_unicode:"&#9821;")
+    Bishop.create(x_position: 5, y_position: 0, game_id: id, user_id: black_user_id, piece_unicode:"&#9821;")
 
-    Queen.create(x_position: 3, y_position: 0, game_id: id, user_id: black_user_id)
-    King.create(x_position: 4, y_position: 0, game_id: id, user_id: black_user_id)
+    Queen.create(x_position: 3, y_position: 0, game_id: id, user_id: black_user_id, piece_unicode:"&#9819;")
+    King.create(x_position: 4, y_position: 0, game_id: id, user_id: black_user_id, piece_unicode:"&#9818;")
 
     # white pieces
     (0..7).each do |i|
-      Pawn.create(x_position: i, y_position: 7, game_id: id, user_id: white_user_id)
```

Comment:
This is apparently a bug from before, but the `y_position` for white pawns should be `6` not `7`.